### ISSUE TITLE
0.3.0 broke something on macOS 10.10

### ISSUE THREAD

**Issue by justvanrossum** [robotools/vanilla#187]
In FontGoggles, clicking on the "Variations" tab makes the whole tab move to the bottom of the parent, rendering it unusable.

0.3.0:
![image](https://github.com/robotools/vanilla/assets/4246121/fda471a4-a256-4707-b72e-eb8d11998d32)

0.2.4:
![image](https://github.com/robotools/vanilla/assets/4246121/25522d6e-78b7-429e-a931-857ade8c277d)

This does not seem to happen on macOS 10.15.

It may not be worth fixing this, but I thought I could at least document it.

**Comment by typemytype** [comment]
macOS 10.10 is released in 2014.... 

I guess a more clear statement of vanilla support over different macOS version could help here... 

We should consider that we cannot keep up supporting the latest macOS with losing older macOS versions.

**Comment by typemytype** [comment]
no idea... 

things to consider:
* py3.9 EOL is October 2025
* switch between intel M1
* M1 is only Monterey+

**Comment by typemytype** [comment]
does it happens for each tab view? or only a tab view inside a splitview?

maybe a small example?

the change in vanilla was a move for the tab view to a [NSTabViewController](https://developer.apple.com/documentation/appkit/nstabviewcontroller?language=objc)

**Comment by justvanrossum** [comment]
Hard to say, I but I'll try to make a small reproducer.

**Comment by chrisjansky** [comment]
Related: ```SegmentedButton``` doesn’t respect posSize for me on Mac OS 12.3.1

Screenshot:
<img width="434" alt="image" src="https://github.com/robotools/vanilla/assets/5342612/1f070bc8-9c7a-4c10-9efb-3732a44497f5">

Test case:
```
import vanilla

def run():
    w = vanilla.Window(
        (400, 400),
    )
    w.btn = vanilla.SegmentedButton(
        (10, 10, -10, 20),
        [
            dict(title="Very Long Title"),
            dict(title="Another Long Title"),
            dict(title="Long Title Indeed"),
        ],
    )

    w.open()
    w.center()

run()
```

Same code executed with ```executeVanillaTest``` works fine:
<img width="428" alt="image" src="https://github.com/robotools/vanilla/assets/5342612/06684084-fbc9-4032-a9e2-fea2db7e45d4">

**Comment by typemytype** [comment]
could you test with the latest vanilla from the repo? 
the vanilla version on pypi is behind

**Comment by chrisjansky** [comment]
@typemytype Not sure I follow, I am using vanilla ```v0.3.0``` which looks like the latest version here on GitHub Releases also

**Comment by typemytype** [comment]
true, but the latest commit is newer and already fixed this issue, sorry for being lazy on the releases

`pip3 install git+https://github.com/robotools/vanilla` to install the latest from this repo
